src/views/search/search.jsx

```diff
@@ -0,0 +1,148 @@
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var Api = require('../../mixins/api.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var Tabs = require('../../components/tabs/tabs.jsx');
+var Carousel = require('../../components/carousel/carousel.jsx');
+var offset = 0;
+var more = [];
+var tab = 'projects';
+var searchTerm = '';
+
+require('./search.scss');
+
+var Search = injectIntl(React.createClass({
+    type: 'Search',
+    mixins: [
+        Api
+    ],
+    getInitialState: function () {
+        return {};
+    },
+    componentDidMount: function () {
+        var pathname = window.location.search;
+        var q = pathname.lastIndexOf('q=');
+        if (q!=-1) {
+            searchTerm = pathname.substring(q+2,pathname.length).toLowerCase();
+        }
+        while (searchTerm.indexOf('/')>-1) {
+            searchTerm = searchTerm.substring(0,searchTerm.indexOf('/'));
+        }
+        while (searchTerm.indexOf('&')>-1) {
+            searchTerm = searchTerm.substring(0,searchTerm.indexOf('&'));
+        }
+        searchTerm = searchTerm.split('+').join(' ');
+        this.getSearchResults(0);
+    },
+    getSearchResults: function () {
+        var termText = '';
+        if (searchTerm!='') {
+            termText = '&q='+searchTerm;
+        }
+        this.api({
+            uri: '/search/projects?limit=16'+termText
+        }, function (err, body) {
+            if (!err) this.setState({searchResults: body});
+        }.bind(this));
+    },
+    getSearchMore: function () {
+        var termText = '';
+        if (searchTerm!='') {
+            termText = '&q='+searchTerm;
+        }
+        offset+=16;
+        this.api({
+            uri: '/search/projects?limit=16&offset='+offset+termText
+        }, function (err, body) {
+            if (!err) this.setState({searchMore: body});
+        }.bind(this));
+    },
+    renderRows: function () {
+        var row2 = this.state.searchResults;
```

Comment:
Remove `carousel` and replace with new component called `grid` that accepts an array of projects.